lib/much-rails/wrap-and-call-method.rb

```diff
@@ -0,0 +1,36 @@
+require "much-plugin"
+require "much-rails/call-method"
+require "much-rails/wrap-method"
```

Comment:
```suggestion
require "much-rails/call_method"
require "much-rails/wrap_method"
```
Use underscores in file names (except for `much-rails` b/c that's the name of the gem). Update everywhere.